app/src/main/kotlin/compiler/ast/AstNode.kt

```diff
@@ -30,6 +30,7 @@ sealed interface AstNode {
                         if (this.initializationType == Expression.ArrayAllocation.InitializationType.ONE_VALUE) arrayValues += "..."
                         "ciąg ${this.elementType}[${this.size.toSimpleString()}]{ $arrayValues }"
                     }
+                    is Expression.ArrayGeneration -> "ułożenie ${this.generatorCall.toSimpleString()}"
```

Comment:
This can lead to weird error messages when ciąg-comprehension (that appears in compiled code) will be shown as generator call (of some dummy generator), but I guess we can't do anything about it (at least without more serious changes in the project).